frontend/src/modules/entitydetails/components/EntityDetails.test.js

```diff
@@ -21,6 +22,8 @@ const ENTITIES = [
                 warnings: [],
             },
         ],
+        project: { contact: '' },
```

Comment:
Same like before, this is to be able to deep render the component tree under test.